feature/floor-map/src/main/java/io/github/droidkaigi/confsched2023/floormap/component/PreviewFloorMapSwitcherFloorLevelProvider.kt

```diff
@@ -0,0 +1,9 @@
+package io.github.droidkaigi.confsched2023.floormap.component
+
+import androidx.compose.ui.tooling.preview.PreviewParameterProvider
+import io.github.droidkaigi.confsched2023.model.FloorLevel
+
+class PreviewFloorMapSwitcherFloorLevelProvider : PreviewParameterProvider<FloorLevel> {
```

Comment:
I like to have this in FloorLevelSwitcher.kt since it is not related to other files. But I'm not that particular about this 👀